specs/tendrl_ansible_grafana_installation.adoc

```diff
@@ -0,0 +1,156 @@
+= Tendrl ansible to install grafana
+
+Needs ansible playbook scripts for automatic installation and configuration of
+grafana and tendrl metrics in the tendrl server.
+
+
+== Problem description
+
+Grafana will be used in tendrl to trigger alerts and visualize the tendrl
+metrics. Currently, Tendrl-ansible is used to install and configure tendrl
+packages. It should also support installation and configuration of tendrl
+metrics and grafana package installation.
+This specification provides the required changes in tendrl-ansible tool to
+support installation and configuration of tendrl metrics and grafana.
+
+
+== Use Cases
+
+* Provide playbook for tendrl-ansible to install tendrl-metrics and grafana
+along with server installation.
+
+
+== Proposed change
+
+* Add the repo to install latest grafana and its dependencies
+
+* Install tendrl metrics, grafana, fontconfig, freetype* and urw-fonts
+
+* Configure tendrl metrics and  grafana
+  This should add the required environment variables, set log path, database
```

Comment:
Details like database connection etc, should they be manually added to playbook? It would be good if you provide a sample playbook and mention the sections which needs to be updated before running it.